eclass/linux-mod-r1.eclass

```diff
@@ -330,6 +331,15 @@ linux-mod-r1_pkg_setup() {
 	debug-print-function ${FUNCNAME[0]} "${@}"
 	[[ ${MERGE_TYPE} != binary ]] || return 0
 	_MODULES_GLOBAL[ran:pkg_setup]=1
+
+	if [[ -z ${ROOT} && ${MODULES_INITRAMFS_IUSE} ]] &&
+		use dist-kernel && use ${MODULES_INITRAMFS_IUSE#+}
+	then
+		# Check, but don't die because we can fix the problem and then
+		# emerge --config ... to re-run installation.
+		nonfatal mount-boot_check_status
```

Comment:
The function itself is provided by `mount-boot-utils.eclass`, I split it out of the `mount-boot.eclass` because that eclass exports a lot of phases which would then also suddenly be set if `linux-mod-r1` is inherited, I expect doing that would break things.

This splitting leaves `mount-boot.eclass` as a "shell" that only inherits `mount-boot-utils.eclass` and then exports the phases.

Reply:
Well, replace what I said by inherit mount-boot-utils then.

Reply:
I already did that on line 141.

Reply:
Oh sorry, I missed that. I ended up checking the main inherit line when I was wondering about it.